openbb_terminal/stocks/stocks_controller.py

```diff
@@ -198,7 +198,7 @@ def call_search(self, other_args: List[str]):
         )
         parser.add_argument(
             "-g",
-            "--industry-group",
+            "--industry_group",
```

Comment:
Is there a reason for reverting this.  Would like to keep a standard of "-" in between multiple inputs.

Reply:
This is to fix the autosuggestion issue. I read up on ArgumentParser and it basically converts any `-` into `_` after the initial `--`. And when I modified the code to this, the autosuggestion started working.
If there's another way to fix this, please let me know. I'd prefer `--industry-group` over `--industry_group`.

Reply:
not following.  if i change to the --exchange-country, this works no problem:

<img width="705" alt="Screenshot 2023-04-21 at 2 13 36 PM" src="https://user-images.githubusercontent.com/18151143/233706580-b28f158e-0773-4328-a274-a7e23f2b2185.png">

Reply:
It seems that the argument names are too similar. I can make it work by changing the name of the argument to, `--ig` and not including `--industry-group` as an argument name.

![Screenshot 2023-04-21 at 2 02 39 PM](https://user-images.githubusercontent.com/85772166/233734196-63368e13-368b-439f-890c-7c6527eaa441.png)

Reply:
That is likely the issue

Reply:
I am fine with renaming args, just not having the _ in the name

Reply:
@the-praxs, the integrated and unit tests will need to be updated and/or re-recorded to reflect changes to the argument name.

Reply:
@jmaslek I will rename the args and substitute _ with - to ensure consistency.

@deeleeramone I will look into this once I have made the changes.